tests/mmm/test_plotting.py

```diff
@@ -108,7 +108,7 @@ class ToyMMM(BaseDelayedSaturatedMMM, MaxAbsScaleTarget):
             ("plot_errors", {"original_scale": True}),
             ("plot_errors", {"ax": plt.subplots()[1]}),
             ("plot_components_contributions", {}),
-            ("plot_channel_parameter", {"param_name": "alpha"}),
+            ("plot_channel_parameter", {"param_name": "adstock_alpha"}),
```

Comment:
We should make sure we are returning just the figure object, otherwise the plots will fail (see comments above)